vueui/src/components/view/cooking/microwave.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div>
+    <template v-if="on == true">
+      <p>Cooking...</p>
+      <vui-progress v-if="cook_time" :value="cur_time" :max="cook_time"/>
+      <vui-button class="danger danger-control" icon="exclamation-triangle" push-state :params="{abort: 1}">Abort!</vui-button>
+    </template>
+    <template v-else>
+      <h4>Ingredients</h4>
+      <ul>
+        <li v-if="cookingobjs.length == 0 && cookingreas.length == 0">The microwave is empty!</li>
```

Comment:
Might want to use computed property, but I am still not sure as this is WIP.
Also migth want to make this statement not inside buletin list.
```suggestion
        <li v-if="!cookingobjs.length && !cookingreas.length">The microwave is empty!</li>
```